src/App.jsx

```diff
@@ -5,14 +5,13 @@ import { PersistGate } from "redux-persist/integration/react";
 import { BrowserRouter as Router, Routes } from "react-router-dom";
 import generateRoutes from "./utils/generateRoutes";
 import routes from "./routes/index";
-import PageOne from "./component/PageOne";
 
 function App() {
   return (
     <>
       <Provider store={store}>
         <PersistGate loading={null} persistor={persistor}>
-          <PageOne />
+          {/* <PageOne /> */}
           <Router>
```

Comment:
remove this commented line. Only comment when you dont want something temporarily

Reply:
Understood will make changes accordingly.